src/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -100,7 +100,8 @@ public void start() {
       addBombardmentSources();
       m_needToAddBombardmentSources = false;
     }
-    m_battleTracker.fightBombingRaids(m_bridge);
+    m_battleTracker.fightAirRaidsAndStrategicBombing(m_bridge);
```

Comment:
I'm confused by this comment. If you merged the methods you would still have the same issue. In fact, if you are arguing that ordering is a code smell how can you stipulate the order?